frontend/src/components/PostContent.js

```diff
@@ -37,44 +38,70 @@ const PostContent = ({ animationDelay, post, review, userMenu }) => {
     ? `/profile`
     : `/authors/${post.author.userName}`
 
+  const handleNavigate = e => {
+    e.stopPropagation()
+    const text = document.getSelection().toString()
```

Comment:
I think you could name it better
```suggestion
    const selectedText = document.getSelection().toString()
```

Reply:
done